diff -burNp geany-0.17/geany.desktop.in geany-0.17-dud/geany.desktop.in
--- geany-0.17/geany.desktop.in	2008-06-06 16:27:08.000000000 +0200
+++ geany-0.17-dud/geany.desktop.in	2009-05-08 06:39:51.877253539 +0200
@@ -3,7 +3,9 @@ Type=Application
 Version=1.0
 _Name=Geany
 _GenericName=Integrated Development Environment
-_Comment=A fast and lightweight IDE using GTK2
+_Comment=A fast and lightweight IDE using GTK+2
+GenericName[pl]=Zintegrowane środowisko programistyczne (IDE)
+Comment[pl]=Szybkie i lekkie IDE używające GTK+2
 Exec=geany %F
 Icon=geany
 Terminal=false
