diff -aurN geany-0.8.orig/src/Makefile.am geany-0.8/src/Makefile.am
--- geany-0.8.orig/src/Makefile.am	2006-08-09 20:15:43.000000000 +0200
+++ geany-0.8/src/Makefile.am	2006-08-12 18:08:22.012690793 +0200
@@ -58,7 +58,7 @@
 AM_CFLAGS = -Wall -pipe
 #AM_CFLAGS = -DGEANY_DEBUG -Wall -pipe -g -O0
 geany_SOURCES = $(SRCS) vte.c vte.h
-geany_LDADD = @PACKAGE_LIBS@ -lstdc++ ../scintilla/libscintilla.a ../tagmanager/libtagmanager.a $(INTLLIBS)
+geany_LDADD = ../scintilla/libscintilla.a ../tagmanager/libtagmanager.a @PACKAGE_LIBS@ -lstdc++ $(INTLLIBS)
 
 INCLUDES = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" -DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
